CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.12.0)
 
 project(ni_grpc_device_server
   LANGUAGES C CXX
-  VERSION 2.8.0)
+  VERSION 2.9.0)
```

Comment:
Make sure to bump the AzDo version first so this update goes into the 2.9 exports on that side of things and not 2.8.

Reply:
Nevermind, checked and it's done on the AzDo side. Thanks!